examples/static_react_task/webapp/src/components/core_components.jsx

```diff
@@ -62,7 +66,9 @@ function SimpleFrontend({ taskData, isOnboarding, onSubmit }) {
             <div className="control">
               <button
                 className="button is-success is-large"
-                onClick={() => onSubmit({ rating: "good" })}
+                // Test for type 2 errors
+                onClick={()=> {throw new Error("test error event_handler");}}
+//                 onClick={() => onSubmit({ rating: "good" })}
```

Comment:
just a reminder to revert this when we're ready to merge

Reply:
to echo Jack, these errors in the callback may actually be caught as type 3 errors, instead of 2

Reply:
the event handlers errors which can't be caught by React's `ErrorBoundary` are caught by the `window.onerror` event handler and those which can be caught by both the `window.onerror` event handler and `ErrorBoundary` are caught twice but this should be the case for `mode=development` and once the `mode=production` it should be caught only once in the `ErrorBoundary` [according to StackOverFlow]